assets/shared/helpers/player/use-editor-player.js

```diff
@@ -162,8 +162,10 @@ const useEditorPlayer = ( videoBlock ) => {
 		const doc = editorCanvasIframe?.contentDocument || document;
 		const w = editorCanvasIframe?.contentWindow || window;
 
-		const isJetpackVideoPress = !! videoBlock.attributes
-			.videoPressClassNames;
+		const isJetpackVideoPress =
```

Comment:
It will make the integration behave the same way it happens with the VideoPress embed.